raiden_contracts/tests/test_channel_settle.py

```diff
@@ -46,107 +47,62 @@ def test_settle_no_bp_success(
 
 def test_settle_channel_state(
         web3,
+        get_accounts,
         custom_token,
         token_network,
-        create_channel,
-        channel_deposit,
-        get_accounts,
-        get_block,
-        create_balance_proof,
-        create_balance_proof_update_signature
+        create_channel_and_deposit,
+        withdraw_channel,
+        close_and_update_channel,
+        settle_state_tests,
+        channel_test_values
```

Comment:
So from what I understand and by reading [this](https://github.com/raiden-network/raiden-contracts/pull/67/files#diff-155b1944c9c54cfb5ecab176aefe584dR11) line I see this is a way to parametrize the test with different values.

It's not really readable, and forces me to go back and read the fixture's definition and then to follow the code upwards to see what's happening. I suggest not having it in a separate fixture that you use only once, but instead to use `pytest.mark.parametrize()` like we do in Raiden. For example [here](https://github.com/raiden-network/raiden/blob/master/raiden/tests/unit/test_binary_encoding.py#L80)